src/auditlog/models.py

```diff
@@ -94,6 +94,9 @@ def get_for_objects(self, queryset):
 
         if isinstance(primary_keys[0], integer_types):
             return self.filter(content_type=content_type).filter(Q(object_id__in=primary_keys)).distinct()
+        elif isinstance(queryset.model._meta.pk, models.UUIDField):
```

Comment:
@jjkester Yes, that is an option. However, I believe `primary_keys[0]` will fire a query, which I would like to avoid. We can get the type of the primary key from `_meta` without having to query a row and inspect its type that way.